java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/handlers/CreateItemHandler.java

```diff
@@ -22,18 +22,17 @@
 @Component
 public class CreateItemHandler extends MenuBaseCommandHandler<CreateItemCommand> {
 
-  private UUID itemId;
-
   public CreateItemHandler(
       MenuRepository menuRepository, ApplicationEventPublisher applicationEventPublisher) {
     super(menuRepository, applicationEventPublisher);
   }
 
   @Override
   Optional<UUID> handleCommand(Menu menu, CreateItemCommand command) {
-    itemId = UUID.randomUUID();
-    Category category = addItem(getCategory(menu, command), command);
+    UUID itemId = UUID.randomUUID();
```

Comment:
I think we can remove this variable and simple set command.setItemID(UUID.randomUUID())